### ISSUE TITLE
[Feature Request] Add mode cycle or previous/next button 

### ISSUE THREAD

**Issue by regraded** [darkreader/darkreader#3290]
Hey gang, the new design looks very nice and polished! I just have a small UI/UX suggestion:

I find myself very frequently switching between theme generator modes (Static, Filter, Filter+, Dynamic) depending on the website. You know, some sites work better with Dynamic, some with Filter etc. 

In the old design, having all the modes side by side means you can flick between the themes very quickly to see which one works best for a given site. 

In the new design, the modes are in a dropdown which means flicking through in quick succession is not really possible. 

A few suggestions as to how to overcome this would be:
- Add next/previous arrows < > on either side of the dropdown
- Add a single 'cycle' button that... cycles through :D
- Display the options as individual buttons (as per the old design)

I'd like to also say thanks for all your work! I'm vision compared and looking at anything on a white background is basically impossible for me so this extension has been a lifesaver and enables me to continue to go about my daily life on the web 😎

**Comment by Gusted** [comment]
Hi!

So, okay let me get this right. You cycle because your on different website right and than go to the right mode for this?

Regards,
Gusted

**Comment by regraded** [comment]
Yes, I will load a website and sometimes it is unreadable due to theme mode (eg black text has been inverted and the site has a light coloured background image which has also been inverted, or perhaps the site is already dark themed and is now being inverted to white) so I will toggle the 'only for x site' option, and then quickly cycle through the modes until I find one that works best for the specific site. 

This is an example of fast cycling through the modes:

![darkmode_cycle](https://user-images.githubusercontent.com/42285123/88993715-08538580-d32a-11ea-8b02-d52748e2959f.gif)

**Comment by Gusted** [comment]
I'm pretty sure if you can use  a shortcut https://github.com/darkreader/darkreader/blob/master/src/manifest.json#L57 it will be faster. You can go the main page and press on set shortcut under toggle site.
